sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -22,7 +22,9 @@ export interface ContextAPI {
 // @public
 export function createSpanFunction(args: CreateSpanFunctionArgs): <T extends {
     tracingOptions?: OperationTracingOptions | undefined;
-}>(operationName: string, operationOptions: T | undefined) => {
+} & {
+    spanOptions?: SpanOptions | undefined;
```

Comment:
I'm a little confused about why we need this - if we're going to move the attribute stuff into context, this goes away, right? Or is this just a point in time state?

Reply:
Yes, but we still need to provide our client libraries a way to customize the newly created span. For example, evenhubs / servicebus would need to pass Link[] as well as SpanKind. We just don't need spanOptions as part of the public API of client libraries

Reply:
Hm I see what you mean.